docs/docs-content/release-notes/known-issues.md

```diff
@@ -0,0 +1,40 @@
+| -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------- | ----------------- |
+| Conducting cluster node scaling operations on a cluster undergoing a backup can lead to issues and potential unresponsiveness.                                                                                                                                                                                                                     | To avoid this, ensure no backup operations are in progress before scaling nodes or performing other cluster operations that change the cluster state                                                                                                                           | April 14, 2024 | Clusters          |
+| Palette automatically creates a security group for worker nodes using the format `<cluster-name>-node`. If a security group with the same name already exists in the VPC, the cluster creation process fails.                                                                                                                                      | To avoid this, ensure that no security group with the same name exists in the VPC before creating a cluster.                                                                                                                                                                   | April 14, 2024 | Clusters          |
+| K3s version 1.27.7 has been marked as _Deprecated_. This version has a known issue that causes clusters to crash.                                                                                                                                                                                                                                  | Upgrade to a newer version of K3s to avoid the issue, such as versions 1.26.12, 1.28.5, and 1.27.11. You can learn more about the issue in the [K3s GitHub issue](https://github.com/k3s-io/k3s/issues/9047).                                                                  | April 14, 2024 | Packs, Clusters   |
```

Comment:
```suggestion
| K3s version 1.27.7 has been marked as _Deprecated_. This version has a known issue that causes clusters to crash.                                                                                                                                                                                                                                  | Upgrade to a newer version of K3s to avoid the issue, such as versions 1.26.12, 1.28.5, and 1.27.11. You can learn more about the issue in the [K3s GitHub issue](https://github.com/k3s-io/k3s/issues/9047) page.                                                                  | April 14, 2024 | Packs, Clusters   |
```